README.md

```diff
@@ -98,31 +98,77 @@ cert-manager-webhook-68d464c8b-hvpf6       1/1     Running   0          33s
 
 ### Install KintoHub
 
-Run
+Run the following steps depending on the setup you want.  
+Each one of the following steps are "incremental", you must run all the steps prior to the setup your chose.
```

Comment:
"Each of the following steps is "incremental", you must run all the steps prior to the setup you chose."